src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
 public class SalaryInfo {
+    private static final String DATE_FORMAT = "dd.MM.yyyy";
+    private final SimpleDateFormat dateFormat = new SimpleDateFormat(DATE_FORMAT);
```

Comment:
If you create a formatter, make it a constant field. That will make your code easier to understand. Plus if you use it in several places in your code you will be able to change its value easily with one action(change variable declaration). In other cases, you need to update the value in each place it's being used.